### PR TITLE
Disabled and error font color for inputs

### PR DESCRIPTION
**Pull request by chromaticWaster**
### Description:

- Added CSS variables for disabled font color and error font color.
- Updated shared disabled stories to have values (can be removed).

### Disabled stories 

**Currency field**

Disabled Currency field with value

**Number field** 

Disabled number field

**Email field**

Disabled Email Field

### All Submissions:

* [x] I have followed the guidelines in the Contributing document.
* [x] I have checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change.

### Changes to Core Features:

* [x] I have added an explanation of what my changes do and why I would like to include them.
* [x] I have written new tests for my core changes, as applicable.
* [x] I have successfully ran tests with my changes locally.
* [x] I have added/updated docs, as applicable.

### CODE REVIEW

**Review by BOTLANNER — APPROVED** [review]

File: src/email-field/EmailField.stories.ts [hunk 1]
```diff
@@ -69,10 +69,10 @@ export const Hint = HintStory<EmailField, BaseArgs>('omni-email-field');
 
 export const Error_Label = ErrorStory<EmailField, BaseArgs>('omni-email-field');
 
-export const Value = ValueStory<EmailField, BaseArgs>('omni-email-field');
+export const Value = ValueStory<EmailField, BaseArgs>('omni-email-field', 'JohnDoe@gmail.com');
```
Comment (stefan505) [inline src/email-field/EmailField.stories.ts:72]: Lower case?

File: src/pin-field/PinField.stories.ts [hunk 2]
```diff
@@ -108,4 +108,4 @@ export const Prefix = PrefixStory<PinField, BaseArgs>('omni-pin-field');
 
 export const Suffix = SuffixStory<PinField, BaseArgs>('omni-pin-field');
```
Comment (stefan505) [inline src/pin-field/PinField.stories.ts:110]: 1234? Pin numbers are normally 4 digits min in length.

**Review by stefan505 — COMMENTED** [review]
Check comments on defaults for pin and email. Fine otherwise.

File: src/pin-field/PinField.stories.ts (hunk 2 above)
Comment (chromaticWaster) [reply src/pin-field/PinField.stories.ts:110]: updated to 4 digits

File: src/email-field/EmailField.stories.ts (hunk 1 above)
Comment (chromaticWaster) [reply src/email-field/EmailField.stories.ts]: updated accordingly

**Review by stefan505 — APPROVED** [review]

**Pull request merged**